### ISSUE TITLE
chinese:计算IOU公式错误

### ISSUE THREAD

**Issue by navy63** [kkweon/UNet-in-Tensorflow#11]
这个计算IOU的公式会增加IOU的概率；
正确的公式应该是
intersection = tf.reduce_sum(pred_flat * true_flat, axis=1) + 1e-7
    denominator = tf.reduce_sum(
        pred_flat, axis=1) + tf.reduce_sum(
            true_flat, axis=1) + 1e-7

    return tf.reduce_mean(intersection / (denominator-intersection))
